python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -26,7 +26,6 @@
 import io
 from contextlib import redirect_stdout
 
-from pyspark import StorageLevel
```

Comment:
Just a question. Is this a relevant change?

Reply:
it is not related, it is a not-used import. since we are touching this file, what about also removing it btw?